nilearn/tests/test_masking.py

```diff
@@ -511,25 +520,34 @@ def test_compute_multi_epi_mask():
     assert_array_equal(mask_ab, get_data(mask_ab_))
 
 
-def test_compute_multi_gray_matter_mask():
-    pytest.raises(TypeError, compute_multi_gray_matter_mask, [])
+def test_compute_multi_brain_mask():
+    pytest.raises(TypeError, compute_multi_brain_mask, [])
 
     # Check error raised if images with different shapes are given as input
-    imgs = [Nifti1Image(np.ones((9, 9, 9)), np.eye(4)),
-            Nifti1Image(np.ones((9, 9, 8)), np.eye(4))]
-    pytest.raises(ValueError, compute_multi_gray_matter_mask, imgs)
+    imgs = [data_gen.generate_mni_space_img(res=8, random_state=0)[0],
+            data_gen.generate_mni_space_img(res=12, random_state=0)[0]]
+    pytest.raises(ValueError, compute_multi_brain_mask, imgs)
 
     # Check results are the same if affine is the same
-    rng = np.random.RandomState(42)
-    imgs1 = [Nifti1Image(rng.standard_normal(size=(9, 9, 9)), np.eye(4)),
-             Nifti1Image(rng.standard_normal(size=(9, 9, 9)), np.eye(4))]
-    mask1 = compute_multi_gray_matter_mask(imgs1)
+    imgs1 = [data_gen.generate_mni_space_img(res=9, random_state=42)[0],
+             data_gen.generate_mni_space_img(res=9, random_state=42)[0]]
+    imgs2 = [data_gen.generate_mni_space_img(res=9, random_state=42)[0],
```

Comment:
here all the images will be identical, I think you want to use different seeds: 0, 1, 2, 3 instead of 42, 42, 42, 42